@lilitkarapetyan - the build is failing with this:
```
-----> Build
       Detected both "build" and "heroku-postbuild" scripts
       Running heroku-postbuild (yarn)
       yarn run v1.22.21
       $ yarn build && yarn build-storybook
       $ cd client && yarn build
       $ rimraf build && vite build && yarn copy-assets && yarn copy-assets-storybook && yarn cleanup
failed to load config from /tmp/build_ac65015b/client/vite.config.ts
error during build:
Error: Cannot find module 'vite-plugin-node-polyfills'
Require stack:
- /tmp/build_ac65015b/client/vite.config.ts
- /tmp/build_ac65015b/client/node_modules/vite/dist/node/chunks/dep-3b8eb186.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1134:15)
    at Module._load (node:internal/modules/cjs/loader:975:27)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/tmp/build_ac65015b/client/vite.config.ts:44:41)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at _require.extensions.<computed> [as .js] (file:///tmp/build_ac65015b/client/node_modules/vite/dist/node/chunks/dep-3b8eb186.js:66199:24)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
error Command failed with exit code 1.
       info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
       info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
       info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
-----> Build failed
       
       We're sorry this build is failing! You can troubleshoot common issues here:
       https://devcenter.heroku.com/articles/troubleshooting-node-deploys
```

--- a/client/package.json
+++ b/client/package.json
@@ -133,6 +133,7 @@
     "uniqid": "^5.0.3",
     "universal-router": "^8.1.0",
     "vite": "^4.4.7",
+    "vite-plugin-node-polyfills": "^0.19.0",
     "vite-plugin-babel-macros": "^1.0.6",
     "vite-plugin-checker": "^0.6.1",
     "vite-plugin-env-compatible": "^1.1.1",
@@ -165,7 +166,6 @@
     "stylelint": "^13.13.1",
     "stylelint-config-prettier": "^8.0.2",
     "stylelint-prettier": "^1.2.0",
-    "vite-plugin-node-polyfills": "^0.19.0",
     "webpack": "^5.82.1"
   },
   "scripts": {

--- a/client/src/Components/local/organisms/setting-forces/index.tsx
+++ b/client/src/Components/local/organisms/setting-forces/index.tsx
@@ -14,6 +14,7 @@ import RolesAccordion from './settings-force-roles'
 import styles from './styles.module.scss'
 import PropTypes, { ForceData } from './types/props'
 import { resolveImagePath } from 'src/Helpers'
+
 export const SettingForces: React.FC<PropTypes> = ({
   forces: initialForces,
   onSave,

